.gitattributes

```diff
@@ -0,0 +1,3 @@
+# .ambr files should be excluded from repo stats.
+# https://github.com/github-linguist/linguist/blob/main/docs/overrides.md
+*.ambr linguist-vendored
```

Comment:
🤔  I'm not 100% sure I've done this correctly. I'm trying to avoid giving myself a 20_000 line contribution for this generated file. 

I chose `linguist-vendored` over `linguist-generated` since `linguist-generated` is "Excluded from stats, hidden in diffs", and we don't want to hide these files from diffs.